### ISSUE TITLE
Manually attach to process using sudo/root

### ISSUE THREAD

**Issue by thomasgalliker** [SuessLabs/VsLinuxDebug#39]
Unfortunately, I cannot use VsLinuxDebug to build, deploy and attach. Build and deploy works fine but it does not attach automatically and it's quite frustrating since I don't see any error message in the output window.

Your help is very much appreciated. I have to run several commands in my C# console app under administrative rights, e.g. modify the wpa_supplicant configuration etc... Therefore, I have to launch my console app using "sudo dotnet consoleapp.dll". Whenever I want to attach the debugger, It complains it also requires root previleges in order to attach.

**Comment by DamianSuess** [comment]
Thank you @thomasgalliker for reaching out. To assist:
* Which distribution/version of Linux are you using?
* Which version of .NET are you using?

In the meantime, as a test, could you try the following? And since this is a console app, Step 1 is just needed to assist with allowing the app to start and wait for the remaining manual steps.

1. Add a `Console.ReadLine();` at the beginning of `Main()` and a breakpoint on a line of code right after the `ReadLine()`
2. Follow the, [Debugging Manually](https://github.com/SuessLabs/VsLinuxDebug/wiki/Debugging-Manually) to deploy and _Attach to Process_.

In version 2.0, the main drive is debugging stability. It's known that some distributions or configurations can be troublesome for a fully automated debugging process. Though you're having an issue, your assistance is greatly appreciated to ensure project stability.

Thank you!

**Comment by thomasgalliker** [comment]
Thanks for your help, @DamianSuess. I found the problem by manually issuing the commands I found in launch.json. The problem was, that putty (and plink) complained about an updated fingerprint. This is probably an edge case scenario but it can happen if I setup my linux (raspian) image several times and try to commect to it with the same IP.

Maybe there is a way to omit or accept this fingerprint change in plink.exe?

I noted some steps in a word doc how I got it up and running. I try to re-setup my raspi again and then I can send you my step-by-step guide. It also contains some manual steps to run vsdbg with sudo. Pretty cool stuff.

**Comment by thomasgalliker** [comment]
@DamianSuess no, that's what I' was thinking too. I cleared the old fingerprint from known_hosts but the problem still exists. Next, I tried to run the plink.exe with all the arguments from launch.json in a new command line window.
Putty/plink store the fingerprint in registry keys... `HKEY_CURRENT_USER\Software\SimonTatham\PuTTY\SshHostKeys`

**Comment by DamianSuess** [comment]
Thank you for the feedback, Thomas.

I'm with you, the extension should be able to react to this kind of error informing the user. It's assigned to the backlog for the next release. If you're feeling ambitious, PRs are always welcome.

Thanks again for supporting the extension. 👍 Hope you have a wonderful week ahead!
